modules/EvseManager/CMakeLists.txt

```diff
@@ -21,6 +21,10 @@ target_sources(${MODULE_NAME}
         PersistentStore.cpp
 )
 
+target_include_directories(${MODULE_NAME} PRIVATE
+    ../../lib/staging/util
```

Comment:
why don't we just link against the lib/stating/util?

Reply:
> why don't we just link against the lib/stating/util?

because there are no object files to link against. EnumFlags.hpp is a header file only.

Reply:
@a-w50 thank you for the useful information. I've not heard the term header-only library. I'm still relatively new to cmake (used make for 30+ years) so your help is much appreciated.

Reply:
I will add a commit here

Reply:
@james-ctc I added a commit as a draft on how to include additional libraries (in this case `everest::staging::util`) including the test via cmake.
It is working now, but I'm not really happy with the namespacing/naming so far.  The namespace is quite long.  Maybe we can reduce this a bit by using `ev` instead of `everest` and `staging` could become `draft`.  And once it becomes stable it might go to `ev::std::util` or something similar.  Accordingly the include directory structure would need to be adopted.
For now the commit could be taken as is, but soon we should come up with some naming conventions.

Reply:
thank you for your help.